packages/rum/src/domain/record/privacy.spec.ts

```diff
@@ -389,3 +395,26 @@ describe('shouldMaskNode', () => {
     })
   })
 })
+
+describe('serializeAttribute ', () => {
```

Comment:
Could you take some time to move `serializeAttribute` function and tests in `serialize.ts`? i think it would be better suited than `privacy.ts`, especially with this improvement where it is not really related to privacy.

Reply:
Yeah this should be cleaned up. Will tackle in a separate cleanup PR next.

Reply:
Addressed in PR https://github.com/DataDog/browser-sdk/pull/1033